client/extensions/woocommerce/app/order/order-customer/dialog.js

```diff
@@ -280,13 +296,17 @@ class CustomerAddressDialog extends Component {
 export default connect(
 	state => {
 		const address = getStoreLocation( state );
+		const locationsLoaded = areLocationsLoaded( state );
 		const areSettingsLoaded = areSettingsGeneralLoaded( state );
+		const countries = getAllCountries( state );
 
 		return {
+			areLocationsLoaded: locationsLoaded,
```

Comment:
`propTypes` :)

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/22838/commits/705b284462b5390c6fa399f6c7eaef08f0cc96a9